src/colors/color.rs

```diff
@@ -337,6 +337,48 @@ impl Color {
         (long, medium, short)
     }
 
+    /// convert normalized RGB value to linear value
+    fn to_linear(u: f32) -> f32 {
+        if u >= 0.04045 {
+            ((u + 0.055) / (1. + 0.055)).powf(2.4)
+        } else {
+            u / 12.92
+        }
+    }
+
+    /// Convert color to OKLAB color space
+    ///
+    /// https://bottosson.github.io/posts/oklab/
+    pub fn to_oklab(self) -> (f32, f32, f32) {
+        let red = Self::to_linear(self.red as f32 / 255f32);
+        let green = Self::to_linear(self.green as f32 / 255f32);
+        let blue = Self::to_linear(self.blue as f32 / 255f32);
+
+        let l = 0.4122214708 * red + 0.5363325363 * green + 0.0514459929 * blue;
+        let m = 0.2119034982 * red + 0.6806995451 * green + 0.1073969566 * blue;
+        let s = 0.0883024619 * red + 0.2817188376 * green + 0.6299787005 * blue;
+
+        let l_ = l.cbrt();
+        let m_ = m.cbrt();
+        let s_ = s.cbrt();
```

Comment:
I think it may be better to use another variable name like `l_prime`, etc. `_` is used in [Rust to denoted unused values](https://doc.rust-lang.org/stable/nightly-rustc/rustc_lint_defs/builtin/static.UNUSED_VARIABLES.html), so this might be confusing when glancing over it.

Reply:
changed all such variables to l_aux, m_aux, s_aux